### ISSUE TITLE
OneClassSVM object has no attribute '_n_support'

### ISSUE THREAD

**Issue by SvenMantowsky** [giacomobartoli/core50_segmentation#1]
Hi, 
i tried your code to get the segemtation labels for the core50 dataset and ran into an error. I think the reason is the newer sklearn version i use. 
Can you provide the version you used for developing the code?

Thanks in advance!

**Comment by giacomobartoli** [comment]
@SvenMantowsky which Python version are you using?

**Comment by SvenMantowsky** [comment]
python 3.6.5 - i tested a few versions of scikit-learn downwards from 0.24 on and version 0.21.3 worked now. now I'm getting first results and just need to check if the "delete_background_multimages.py" works fine.
